settings.py

```diff
@@ -79,6 +79,25 @@
 # Set to True if we're allowed to use X-SENDFILE.
 XSENDFILE = False
 
+
+if DEBUG:
+    # To activate the Django debug toolbar.
+    INSTALLED_APPS += (
+        'debug_toolbar',
+    )
+    # Prevent DDT from patching the settings.
+    DEBUG_TOOLBAR_PATCH_SETTINGS = False
+
+    MIDDLEWARE_CLASSES += ('debug_toolbar.middleware.DebugToolbarMiddleware',)
+
+    def show_toolbar_callback(request):
+        return True
+
+    DEBUG_TOOLBAR_CONFIG = {
+        'SHOW_TOOLBAR_CALLBACK': 'local_settings.show_toolbar_callback'
```

Comment:
Nope, according to the docs: `This is the dotted path to a function used for determining whether the toolbar should show or not`